config/7.3.0/fapi1part2adv/ig/routes/routes-service/99-platform-passthrough.json

```diff
@@ -1,8 +1,7 @@
 {
-  "name": "29 - AS pass-through",
-  "comment": "Passthrough for any unprotected AM endpoints (such as the XUI)",
+  "name": "99 - Identity Platform pass-through",
+  "comment": "Pass-through for any unprotected Identity Platform endpoints (such as the XUI) - the name of this route needs to be last alphabetically so that more specific routes can be used if they exist.",
   "baseURI": "https://&{identity.platform.fqdn}",
-  "condition": "${find(request.uri.path, '^/am')}",
```

Comment:
Condition has been removed, ALL unrouted requests will therefore be forwarded to Identity Platform.

This route appears last in the IG route list and will act as the default.